test/support/helpers/database.helper.js

```diff
@@ -32,7 +32,7 @@ async function _tableNames (schema) {
     .select('tablename')
     .where('schemaname', schema)
 
-  return result.map((table) => `${schema}.${table.tablename}`)
+  return result.map((table) => `"${schema}".${table.tablename}`)
```

Comment:
```suggestion
  return result.map((table) => {
    return `"${schema}".${table.tablename}`
  })
```

Reply:
You had mentioned that and I forgot about it. Good spot!